packages/dds/tree/src/core/tree/anchorSet.ts

```diff
@@ -131,6 +131,28 @@ export interface AnchorEvents {
 	 */
 	subtreeChanging(anchor: AnchorNode): PathVisitor | void;
 
+	/**
+	 * Emitted after the subtree rooted at `anchor` may have been changed.
+	 * While this event is always emitted in the presence of changes to the subtree,
+	 * it may also be emitted even though no changes have been made to the subtree.
+	 * May be emitted multiple times within the application of a single edit or transaction.
```

Comment:
Nit: These notes feel like they belong in a `@remarks` block, rather than being a part of the semantic summary.
```suggestion
	 * 
	 * @remarks
	 * While this event is always emitted in the presence of changes to the subtree,
	 * it may also be emitted even though no changes have been made to the subtree.
	 * May be emitted multiple times within the application of a single edit or transaction.
```

Reply:
Good call